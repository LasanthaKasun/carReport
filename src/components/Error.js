import React from "react";
const Error = ({ message }) => {
  return <span className="text-red-500 font-medium text-sm">{message}</span>;
};

export default Error;
