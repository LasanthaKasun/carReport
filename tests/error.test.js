import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import Error from "../src/components/Error";

it("should render error component", () => {
  const component = renderer.create(<Error />);
  expect(component).toMatchSnapshot();
});