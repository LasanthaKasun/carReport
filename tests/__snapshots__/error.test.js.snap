// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render damage report create page 1`] = `
<span
  className="text-red-500 font-medium text-sm"
/>
`;

exports[`should render error component 1`] = `
<span
  className="text-red-500 font-medium text-sm"
/>
`;
